R/lifecycle_admiral.R

```diff
@@ -2,14 +2,13 @@
 #'
 #' Wrapper around `lifecycle::deprecate_soft()`.
 #'
+#' @param when See documentation for `lifecycle::deprecate_soft()`.
+#' @param what See documentation for `lifecycle::deprecate_soft()`.
+#' @param with See documentation for `lifecycle::deprecate_soft()`.
+#' @param details See documentation for `lifecycle::deprecate_soft()`.
+#' @param id See documentation for `lifecycle::deprecate_soft()`.
 #' @param env See documentation for `lifecycle::deprecate_soft()`.
```

Comment:
We can add `#' @inheritParams lifecycle::deprecate_soft()` and roxygen will insert the param descriptions for us. 🕺🏼

Reply:
Doesn't seem to like that - but I want it! 

![image](https://github.com/user-attachments/assets/50baa2be-d5fb-4281-b4b3-5ca8202f719d)

Reply:
Ahhh, that is because I sent you the wrong code 😬 it's meant to be `@inheritParams lifecycle::deprecate_soft` (without the parentheses). I pushed the update FYI

Reply:
very cool!!  thanks for the help!